src/OrchardCore/OrchardCore.ContentLocalization.Core/Handlers/LocalizationPartHandler.cs

```diff
@@ -0,0 +1,84 @@
+using System.Globalization;
+using System.Threading.Tasks;
+using OrchardCore.ContentLocalization.Models;
+using OrchardCore.ContentManagement;
+using OrchardCore.ContentManagement.Handlers;
+using OrchardCore.Entities;
+using OrchardCore.Settings;
+
+namespace OrchardCore.ContentLocalization.Handlers
+{
+    public class LocalizationPartHandler : ContentPartHandler<LocalizationPart>
```

Comment:
I don't understand why you have a handler for this, I would assume assigning the culture is done after the content is cloned and it's localized. Because at that point you have a `LocalizationPart` you can just assign the default culture there, if no specific culture was specified.

Reply:
I think the handler was for when a content is not localized but you attach a localization part to it.
I will try to see if It breaks.